### DISCUSSION TITLE
Getting vs-code hints in English

### DISCUSSION THREAD

**Discussion by adarshmadrecha** [youzan/vant#10184] [Q&A]
As of version `3.`, when we hover on the Component Props, it shows hints in `zh-cn` Language. How do it configure it to use `en-us` Language? 

![image](https://user-images.githubusercontent.com/11911938/149461699-66a4d667-edbd-43c2-a1b6-d3310a957d28.png)

**Comment by chenjiahan** [comment]
What VS Code plugin do you use?

**Reply by adarshmadrecha** [reply]
Vuetr, Tailwind AutoComplete, Vue snipets by Sarah Dagner
